examples/start_sps.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+set -e
+
+# Starts 3 storage providers — Alice, Bob & Charlie
+# Ports start at 45000 and increment 1000 for each, so 45000, 46000 & 47000
+# You can check the configuration in their respective folders
+#
+# It also sets up balances for all accounts — Alice, Bob, Charlie, Dave, Eve & Ferdie
+
+trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
+
+# requires the testnet to be running!
+export DISABLE_XT_WAIT_WARNING=1
+
+mkdir -p /tmp/polka-storage-provider
+
+P2P_ADDRESS="/ip4/127.0.0.1/tcp/62649"
```

Comment:
```suggestion
P2P_BOOTSTRAP_ADDRESS="/ip4/127.0.0.1/tcp/62649"
```
Just to be more explicit.